lm75bd/lm75bd.h

```diff
@@ -5,7 +5,7 @@
 #include <stdint.h>
 
 /* LM75BD I2C Device Address */
-#define LM75BD_OBC_I2C_ADDR /* Define the address here */
+#define LM75BD_OBC_I2C_ADDR 0b1001111U/* Define the address here */
```

Comment:
Hexadecimal is usually used for readability purposes

Reply:
changing to -> #define LM75BD_OBC_I2C_ADDR 0x4FU